code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5287,3 +5287,25 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 		var/mob/living/carbon/human/H = M
 		if(H.species.name == "Vox")
 			M.adjustToxLoss(-4 * REM) //chicken and gravy just go together
+
+/datum/reagent/blockizine
+	name = "Blockizine"
+	id = BLOCKIZINE
+	description = "Some type of material that preferentially binds to all possible chemical receptors in the body, but without any direct negative effects."
+	reagent_state = LIQUID
+	custom_metabolism = 0
+	color = "#B0B0B0"
+
+/datum/reagent/blockizine/on_mob_life(var/mob/living/carbon/human/H)
+	if(..())
+		return 1
+	if(!data)
+		data = world.time+3000
```

Comment:
Eh, it means that blockizine does nothing special at higher amounts, but I guess so.